### PR TITLE
Prepare for ML model switch

### PR DESCRIPTION
**Pull request by samaloney**
Remove old mock API code based on SRS regions and prepare for ML model inference

### CODE REVIEW

File: app/classify.py [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
-    tend = time + 12 * u.hour
-    srs_query = Fido.search(a.Time(tstart, tend), a.Instrument.soon)
-    if 'srs' not in srs_query.keys():
-        raise SRSNotFoundError(f'No SRS file found for {time.isot}')
-    closest_idx = np.argmin(srs_query['srs']['Start Time'] - time)
-    srs_file = Fido.fetch(srs_query['srs'][closest_idx])
-    if len(srs_file.errors) != 0:
-        raise SRSDownloadError(f"Error downloading file {srs_query['srs']['url']}")
-    return srs_file
+    query = Fido.search(a.Time(time, time + timedelta(minutes=1)),
```
Comment (samaloney) [inline app/classify.py:24]: Might need to switch to JSOC/DRMS as can get different series here e.g. 45s v 12minute but for the moment should be ok.

File: app/classify.py [hunk 2]
```diff
@@ -51,70 +51,59 @@ def classify(time: datetime, hgs_latitude: float, hgs_longitude: float):
...
+    """
+    mag_map = download_magnetogram(time)
+    mag_map.data
+    # ML model here
+
+    detections = []
+    detection = {'time': time,
+                 "bbox": {"bottom_left": {"latitude": "", "longitude": ""},
+                          "top_right": {"latitude": "", "longitude": ""}},
+                 "hale_class": "", "mcintosh_class": ""}
```
Comment (samaloney) [inline app/classify.py:97]: Should maybe return all the classes and prob/logits as well

**Review by edoardolegnaro — APPROVED** [review]
Looks good
